src/patternfly/components/Form/form.scss

```diff
@@ -1,3 +1,29 @@
+$pf-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "lg", "xl", "2xl");
+
+// updates the mixin name to something more unique to avoid namespace conflicts
```

Comment:
Can you remove these comments?

Reply:
removed :) Thanks for the patience reviewing this!